autoload/lsp.vim

```diff
@@ -390,6 +390,9 @@ function! lsp#default_get_supported_capabilities(server_info) abort
     \              'valueSet': lsp#omni#get_completion_item_kinds()
     \           }
     \       },
+    \       'definition': {
```

Comment:
Will this PR also work for type definition, declaration and implementation? If so, we can expand the capabilities.

Reply:
see above :)

(spoiler: yes, but I can't test that.)

Reply:
I don't know a language server that implements both type def and location link, either, unfortunately. I suppose we can just add the capabilities. I don't see any reason why definition would work but e.g. type def won't.